website/src/content/docs/docs/howtos/Generate client libraries/02client.mdx

```diff
@@ -67,6 +67,74 @@ client.pet();
 
 </ClientTabs>
 
+### Clients from sub namespaces
+
+<ClientTabs>
+
+```typespec title="main.tsp"
+namespace PetStore;
+
+@service({
+  title: "Dog Store",
+})
+namespace Dogs {
```

Comment:
This is interesting, because your example is two servies in one TypeSpec. Which I didn't even think we supported, so if we do we should keep it for sure :)

I was thinking more about sub-namespaces, like in todo app:
https://github.com/allenjzhang/typespec-e2e-demo/blob/main/todoApp/spec/main.tsp

Meaning, one service, and sub-namespaces are operation groups (or subclient)

TLDR: If we support two services, great let's keep that. But then add also example with sub-namespaces and operation groups / sub client.

Reply:
weidong has reminded me about multiple service limitation. tcgc used to report warning for this, but could give the correct type graph. let me check with all languages to see if it works.
the doc already has the case of operation group with sub namespace. but since sub client from operation group could only get from parent, current language usage code could not show the difference for sub client in sub namespace. do you think we should add the generated code to the doc to show such feature?